cmd/lncli/commands.go

```diff
@@ -331,16 +332,26 @@ var sendCoinsCommand = cli.Command{
 				"scripts",
 		},
 		coinSelectionStrategyFlag,
+		cli.StringSliceFlag{
+			Name: "utxo",
+			Usage: "a utxo specified as outpoint(tx:idx) which " +
+				"will be used as input for the transaction " +
+				"The selected utxos can either be entirely " +
+				"spent by specifying the sweepall flag or a " +
+				"specified amount can be spent in the utxo " +
+				"through the amt flag",
```

Comment:
We should also mention that this flag can be used multiple times in the same command, because on the command line help you don't see that it's of type `StringSliceFlag`